remote-trait-object-macro/src/helper/dispatcher.rs

```diff
@@ -184,9 +184,9 @@ pub fn generate_dispatcher(source_trait: &syn::ItemTrait) -> Result<TokenStream2
                     #if_else_clauses_rwlock
                 }
             }
-            impl #env_path::ToDispatcher<dyn #trait_ident> for Box<dyn #trait_ident> {
-                fn to_dispatcher(self) -> std::sync::Arc<dyn #env_path::Dispatch> {
-                    std::sync::Arc::new(#box_dispatcher_ident::new(self))
+            impl #env_path::IntoService<dyn #trait_ident> for Box<dyn #trait_ident> {
+                fn into_service(self) -> #env_path::ServiceToRegister {
```

Comment:
I have a concern that `into_service` can make users confusing about the  `service` concept.
Anyway, it seems better than `to_dispatcher`.

Reply:
Let's use `service` for server side only.